### PR TITLE
Web/add dataset drawer

### PR DESCRIPTION
**Pull request by davidsharp7**
### Problem

An idea to move some column level attributes to a side drawer to make better use of screen real estate and with a view of adding additional column level information.

Screenshot 2023-10-30 at 13 29 41

### Solution

Add a drawer which displays column level information (tags at the moment) on the right hand of the screen when a column is selected.  Use an accordion to allow hide/expand data elements.

One-line summary:

Add drawer to dataset column view.

### Checklist

- [x] You've [signed-off](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#sign-your-work) your work
- [x] Your changes are accompanied by tests (_if relevant_)
- [x] Your change contains a [small diff](https://kurtisnusbaum.medium.com/stacked-diffs-keeping-phabricator-diffs-small-d9964f4dcfa6) and is self-contained
- [x] You've updated any relevant documentation (_if relevant_)
- [x] You've included a one-line summary of your change for the [`CHANGELOG.md`](https://github.com/MarquezProject/marquez/blob/main/CHANGELOG.md#unreleased) (_Depending on the change, this may not be necessary_).
- [x] You've versioned your `.sql` database schema migration according to [Flyway's naming convention](https://flywaydb.org/documentation/concepts/migrations#naming) (_if relevant_)
- [x] You've included a [header](https://github.com/MarquezProject/marquez/blob/main/CONTRIBUTING.md#copyright--license) in any source code files (_if relevant_)

### CODE REVIEW

**Comment by davidsharp7** [comment]
@phixMe @wslulciuc - this is what I was talking about.  It's a bit of hack but you get the idea.

File: web/src/components/datasets/DatasetInfo.tsx [hunk 1]
```diff
@@ -95,44 +127,97 @@ const DatasetInfo: FunctionComponent<DatasetInfoProps> = (props) => {
...
+                )
+              })}
+            </TableBody>
+          </Table>
+          <Drawer
+            elevation={0}
+            anchor='right'
+            open={open}
+            onClose={handleClose}
+            sx={{ zIndex: 1300 }}
```
Comment (phixMe) [inline web/src/components/datasets/DatasetInfo.tsx:175]: Can you reference a variable for this in the theme?

Comment (davidsharp7) [reply web/src/components/datasets/DatasetInfo.tsx:175]: yeah cool will move this out.

Comment (davidsharp7) [reply web/src/components/datasets/DatasetInfo.tsx:175]: Cool will fix this up.

**Review by phixMe — APPROVED** [review]
Thnks @davidsharp7

**Pull request merged**